Listing '.'...
Listing './bin'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/django_powerdns_manager-0.2.9a1.egg-info'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/powerdns_manager'...
Listing './lib/python3.5/site-packages/powerdns_manager/fixtures'...
Listing './lib/python3.5/site-packages/powerdns_manager/locale'...
Listing './lib/python3.5/site-packages/powerdns_manager/locale/el'...
Listing './lib/python3.5/site-packages/powerdns_manager/locale/el/LC_MESSAGES'...
Listing './lib/python3.5/site-packages/powerdns_manager/management'...
Listing './lib/python3.5/site-packages/powerdns_manager/management/commands'...
Compiling './lib/python3.5/site-packages/powerdns_manager/management/commands/importaxfr.py'...
***   File "./lib/python3.5/site-packages/powerdns_manager/management/commands/importaxfr.py", line 89
    except Exception, e:
                    ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/powerdns_manager/management/commands/importzones.py'...
***   File "./lib/python3.5/site-packages/powerdns_manager/management/commands/importzones.py", line 72
    except Exception, e:
                    ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/powerdns_manager/migrations'...
Listing './lib/python3.5/site-packages/powerdns_manager/templates'...
Listing './lib/python3.5/site-packages/powerdns_manager/templates/powerdns_manager'...
Listing './lib/python3.5/site-packages/powerdns_manager/templates/powerdns_manager/template'...
Listing './lib/python3.5/site-packages/powerdns_manager/templates/powerdns_manager/zone'...
Listing './lib/python3.5/site-packages/powerdns_manager/templates/powerdns_manager/zone/export'...
Listing './lib/python3.5/site-packages/powerdns_manager/templates/powerdns_manager/zone/import'...
Listing './lib/python3.5/site-packages/powerdns_manager/templatetags'...
Compiling './lib/python3.5/site-packages/powerdns_manager/utils.py'...
***   File "./lib/python3.5/site-packages/powerdns_manager/utils.py", line 230
    except DNSException, e:
                       ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/powerdns_manager/views.py'...
***   File "./lib/python3.5/site-packages/powerdns_manager/views.py", line 85
    except Exception, e:
                    ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './share'...
Listing './share/python-wheels'...
